### PR TITLE
Fix quick find dialog issues

### LINKED ISSUE #1: navigation from the `ctrl + f` search function is really bad
it just tends to bring you to places that aren't actually visible on your screen. bad!

### PR DESCRIPTION
**Pull request by IotaBread**
Fixes the search bar overlapping the matched text, and the "previous" button not wrapping around with the wrap option enabled. Fixes #1

### CODE REVIEW

File: src/main/java/org/quiltmc/syntaxpain/QuickFindDialog.java [hunk 1]
```diff
@@ -95,7 +97,7 @@ public void windowLostFocus(WindowEvent e) {
 				target.requestFocus();
 			}
 		};
-		this.addWindowFocusListener(focusListener);
+		// this.addWindowFocusListener(focusListener);
```
Comment (ix0rai) [inline src/main/java/org/quiltmc/syntaxpain/QuickFindDialog.java:100]: I would rather this just be removed instead of commented

**Review by ix0rai — APPROVED** [review]

File: src/main/java/org/quiltmc/syntaxpain/QuickFindDialog.java (hunk 1 above)
Comment (IotaBread) [reply src/main/java/org/quiltmc/syntaxpain/QuickFindDialog.java:100]: Oh whoops, I commented it out to test without the dialog vanishing

Comment (IotaBread) [reply src/main/java/org/quiltmc/syntaxpain/QuickFindDialog.java:100]: Should I remove it tho?

Comment (ix0rai) [reply src/main/java/org/quiltmc/syntaxpain/QuickFindDialog.java:100]: I think it should stay

Comment (IotaBread) [reply src/main/java/org/quiltmc/syntaxpain/QuickFindDialog.java]: fixed it

**Pull request merged**